sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1477,6 +1515,27 @@ public void close() throws IOException {
       return toBuilder().setProducerFactoryFn(producerFactoryFn).build();
     }
 
+    public Write<K, V> withEOS() {
```

Comment:
<!--new_thread; commit:27f622c4e2c0fabf68d43da4e4aa381f71f82365; resolved:0-->
document what this is